### DISCUSSION TITLE
Export to PDF

### DISCUSSION THREAD

**Discussion by rotter99** [JasonLautzenheiser/trizbort#566] [Issues / Bugs]
V1.7.4.1 When selecting Export to PDF I get the following error: 

There was a problem exporting the map:
Object reference not set to an instance of an object

It does this with both new maps and existing maps.

![Export-to-PDF](https://user-images.githubusercontent.com/98358952/169407923-d43e3044-13fc-48b1-a8da-de04d044de08.png)

.

**Comment by JasonLautzenheiser** [comment]
Just realized I never replied.....yes, the PDF export is broken at this moment.   I'm currently working on getting some of the tech upgraded and reducing some dependencies.   Hope to get the PDF issue fixed as well.  Just been a bigger job making the upgrades than I first anticipated.

**Comment by nil-device** [comment]
I successfully exported to PDF by loading my V1.7.4.1 map in V1.7.2.0. After a warning, it loads the map okay for me and doesn't seem to be missing any info.
